Microsoft.Azure.CosmosRepository/src/Providers/DefaultCosmosContainerDefaultTimeToLiveProvider.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) IEvangelist. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using Microsoft.Azure.CosmosRepository.Builders;
+using Microsoft.Azure.CosmosRepository.Options;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.CosmosRepository.Providers
+{
+
+    /// <inheritdoc/>
+    public class DefaultCosmosContainerDefaultTimeToLiveProvider : ICosmosContainerDefaultTimeToLiveProvider
+    {
+        private readonly IOptions<RepositoryOptions> _options;
+
+        /// <summary>
+        /// Creates an instance of the <see cref="DefaultCosmosContainerDefaultTimeToLiveProvider"/>.
+        /// </summary>
+        /// <param name="options">The repository options.</param>
+        public DefaultCosmosContainerDefaultTimeToLiveProvider(IOptions<RepositoryOptions> options) => _options = options;
+
+        /// <inheritdoc/>
+        public int GetDefaultTimeToLive<TItem>() where TItem : IItem
+        {
+            ContainerOptionsBuilder options = _options.Value.ContainerOptions.FirstOrDefault(co => co.Type == typeof(TItem));
+
+            if (options is null)
```

Comment:
```suggestion
            var options = _options.Value.ContainerOptions?.FirstOrDefault(co => co.Type == typeof(TItem));
            if (options is null)
```

Better safe, than sorry. I'm not sure if this can be `null` or not?

Reply:
Container options are initialized in the class with `new()` so it can never be null.